utils.sh

```diff
@@ -99,6 +99,29 @@ function create_cluster() {
 
     trap auth_template_and_removetmp EXIT
     $OPENSHIFT_INSTALLER --dir "${assets_dir}" --log-level=debug create cluster
+
+    # Workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1816904
+    wait_for_workers
+}
+
+wait_for_workers() {
+    TIMEOUT_MINUTES=30
```

Comment:
In my VM environment there is about 18mins between the node times for masters/workers, so I'm thinking 30+15 is reasonable here - I could also make it configurable but it will be hard for folks to know what values to use...